package.json

```diff
@@ -18,91 +18,57 @@
-    "lint:format": "prettier --check '**/?(.)*.{md,css,scss,js,json,yaml,yml}'",
-    "lint": "npm-run-all -p lint:format lint:js lint:css",
-    "format": "prettier --write '**/?(.)*.{md,css,scss,js,json,yaml,yml}'",
+    "lint:css": "stylelint --report-needless-disables './tbx/static_src/sass' './tbx/static_src/javascript/'",
+    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives .",
+    "lint:format": "prettier --check '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml,vue}'",
+    "lint": "npm run lint:format && npm run lint:js && npm run lint:css",
+    "format": "prettier --write '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml,vue}'",
     "//[ Tests ]//": "",
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "//[ Watches ]//": "",
-    "watch:js": "onchange $npm_package_config_dest_js'/**/**' -- npm-run-all -p reload lint:js",
-    "watch:css": "onchange $npm_package_config_dest_css -- npm-run-all -p reload lint:css",
-    "watch": "npm-run-all -p watch:*",
-    "//[ Tasks ]//": "",
-    "build": "npm-run-all clean compile:dev",
-    "build:prod": "npm-run-all clean compile:prod",
-    "start": "npm-run-all clean compile:dev -p compile:watch watch serve --silent"
+    "//[ Build tasks ]//": "",
+    "build": "webpack --mode development --progress",
+    "build:prod": "webpack --mode production",
+    "start": "webpack --mode development --progress --watch",
+    "start:reload": "webpack-dev-server",
+    "pa11y": "pa11y-ci --sitemap http://localhost:8000/sitemap.xml --config ./pa11y.config.js || (echo 'Please install pa11y-ci globally using: npm install -g pa11y-ci' && exit 1)"
   },
   "devDependencies": {
-    "@babel/core": "^7.16.5",
-    "@babel/preset-env": "^7.16.5",
-    "@babel/preset-react": "^7.16.5",
-    "@babel/eslint-parser": "^7.16.5",
-    "autoprefixer": "^10.4.0",
-    "babel-jest": "^27.4.5",
-    "babel-loader": "^8.2.3",
-    "body-scroll-lock": "^4.0.0-beta.0",
-    "browser-sync": "^2.27.7",
-    "copy-webpack-plugin": "^10.1.0",
-    "css-loader": "5.2.7",
-    "cssnano": "^5.0.13",
-    "eslint": "^8.4.1",
-    "eslint-config-torchbox": "^0.5.0",
-    "file-loader": "^6.2.0",
-    "jest": "^27.4.5",
-    "mini-css-extract-plugin": "^2.4.5",
-    "npm-run-all": "^4.1.5",
-    "onchange": "^7.1.0",
-    "postcss-custom-properties": "^12.0.1",
-    "postcss-loader": "^6.2.1",
-    "prettier": "^2.5.1",
-    "sass": "^1.45.0",
-    "sass-loader": "^12.4.0",
-    "stylelint": "^13.13.0",
-    "stylelint-config-torchbox": "^1.1.1",
-    "url-loader": "^4.1.1",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1"
+    "@types/jest": "^29.5.6",
+    "autoprefixer": "^10.4.16",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "css-loader": "^6.8.1",
+    "cssnano": "^6.0.1",
+    "eslint": "^8.51.0",
+    "eslint-config-torchbox": "^1.1.0",
+    "eslint-webpack-plugin": "^4.0.1",
+    "file-loader": "^5.0.2",
```

Comment:
`file-loader` was deprecated for Webpack v5 so this can go and there's a change needed in `webpack.config` file to switch to `assset-modules` which i'll comment in the file : )

Reply:
https://v5.webpack.js.org/loaders/file-loader/

Reply:
Ah that's odd as I thought I had copied the latest version of package.json and webpack config from kit.

Reply:
Should now be sorted - I committed your change to webpack and removed file-loader